.github/workflows/zap-vs-webseclab.yml

```diff
@@ -0,0 +1,61 @@
+name: ZAP vs Webseclab
+
+on: 
+  schedule:
+    - cron:  '0 4 * * *' # 4 am every day
+  workflow_dispatch:
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone zap-mgmt-scripts and zaproxy-website
+      run: |
+        # Setup git details
+        export GITHUB_USER=zapbot
+        export GITHUB_TOKEN=${{ secrets.ZAPBOT_TOKEN }}
+        git config --global user.email "12745184+zapbot@users.noreply.github.com"
+        git config --global user.name $GITHUB_USER
+        git clone https://github.com/$GITHUB_USER/zap-mgmt-scripts.git
+
+        git clone https://github.com/$GITHUB_USER/zaproxy-website.git
+
+        # Update to the latest upstream
+        cd zaproxy-website
+        git remote set-url origin https://$GITHUB_USER:$GITHUB_TOKEN@github.com/$GITHUB_USER/zaproxy-website.git
+        git remote add upstream https://github.com/zaproxy/zaproxy-website.git
+        git checkout -B webseclab
+        git pull upstream master
+        git reset --hard upstream/master
```

Comment:
I think you may as well just do the reset. No point fetching and merging (pull) just to reset it.

Reply:
Copied from the existing jobs ;)

Reply:
Sorry I missed it :cry: